pkg/apis/multicluster/v1alpha1/gameserverallocationpolicy.go

```diff
@@ -32,9 +32,14 @@ type GameServerAllocationPolicySpec struct {
 
 // ClusterConnectionInfo defines the connection information for a cluster
 type ClusterConnectionInfo struct {
-	ClusterName         string   `json:"clusterName"`
+	// Optional: the name of the targeted cluster
+	ClusterName string `json:"clusterName"`
+	// The endpoint for the allocator service in the targeted cluster
```

Comment:
nit: the comment should be "endpoints" since this is an array. 

As a side note, what is the use case for having more than one endpoint here? Since this is connection info for a single cluster, do we expect to have multiple ways to connect to the allocation endpoint in the cluster? It would be worth updating the comment to explain what we expect in this field and what clients should do if there is more than one entry (and maybe also if it's valid to have an empty array).

Reply:
It looks like from the code / comment below, we currently just walk the list in order (rather than picking a random one each time).

Reply:
Thanks. Updated the comment. 
It is not expected to have multiple endpoints as of now, but it is possible in the future and it follows the [kubernetes ingress model](https://github.com/nginxinc/kubernetes-ingress/blob/master/vendor/k8s.io/api/core/v1/types.go#L3481), which stores the LB IPs as a list.

Reply:
Since this api is still alpha I think this is reasonable, but we should check with the networking folks on k8s whether they think this was a good idea or if still exists for backwards compatibility (e.g. they are stuck with it) before we promote the api to being stable.